src/components/Lend/LoanSearch/ChallengeCallout.vue

```diff
@@ -19,10 +19,11 @@
 							</div>
 							<div class="tw-flex tw-gap-0.5 tw-flex-wrap">
 								<span class="tw-whitespace-nowrap">Added to cart!</span>
-								<template v-if="borrowerName">
-									<span class="tw-whitespace-nowrap">{{ participantsMessage }}</span>
-									<span class="data-hj-suppress tw-whitespace-nowrap">{{ borrowerName }}.</span>
-								</template>
+								<span :class="{'tw-whitespace-nowrap': borrowerName}">{{ participantsMessage }}</span>
+								<span
+									v-if="borrowerName"
```

Comment:
I think we also need to check `this.participants.length > 1`, otherwise the message might not make sense when there's one participant (borrower name just sitting after the `!` in the new message.